src/database/mod.rs

```diff
@@ -1,159 +1,11 @@
-#![cfg(feature = "unstable_db")]
-use crate::app::Hash;
-use clap::Parser;
-use eyre::{eyre, Context, Result};
-use ruint::{aliases::U256, uint};
-use sqlx::{
-    any::AnyKind,
-    migrate::{Migrate, MigrateDatabase, Migrator},
-    pool::PoolOptions,
-    Any, Executor, Pool, Row,
-};
-use tracing::{error, info, instrument, warn};
-use url::Url;
+#[cfg(feature = "unstable_db")]
+mod concrete_database;
```

Comment:
We can make database mandatory. For testing there is the sqlite in-memory database.